src/app/views/home/home.component.ts

```diff
@@ -16,42 +16,34 @@ import {Observable} from 'rxjs';
 })
 export class HomeComponent implements OnInit {
   isEvaluationSaved: boolean;
-  students: Student[];
+  students$: Observable<Student[]>;
   mentor: Mentor;
   mentorStream;
-  constructor(public dialog: MatDialog, private studentService: StudentService,  private mentorService: MentorService,
-              private evaluationService: EvaluationService) { }
+
+  constructor(public dialog: MatDialog, private studentService: StudentService, private mentorService: MentorService,
+              private evaluationService: EvaluationService) {
+  }
 
   ngOnInit(): void {
-    this.getMentorStudents();
+    this.students$ = this.studentService.getMentorStudents().pipe();
```

Comment:
Why did you add `pipe()` here?

Reply:
I did some testing with pipe() method and forgot to delete it. All good now